src/fhirtypes/common.ts

```diff
@@ -486,7 +486,7 @@ export function applyInsertRules(
       logger.error(`Unable to find definition for RuleSet ${rule.ruleSet}.`, rule.sourceInfo);
     }
   });
-  fshDefinition.rules = expandedRules;
+  fshDefinition.rules = cloneDeep(expandedRules);
```

Comment:
I don't think we need to clone the whole `rules` array, as many of those rules don't come from the RuleSet.  I think it is only the rules from the RuleSet that need to be cloned (probably on line 477 above).

Reply:
You're right! I removed the unnecessary copying in the most recent commit, wouldn't want to waste memory.